js/data/locale/currency.json

```diff
@@ -32,7 +32,7 @@
     "KES": {
         "name": "Kenyan Shilling",
         "decimals": 2,
-        "sign": "Sh"
+        "sign": "KES"
```

Comment:
The sign should contain the common use symbols used in the country for that currency, not the ISO code. There is another option on the number formatter to use the ISO code instead.